parsl/executors/workqueue/executor.py

```diff
@@ -191,6 +201,13 @@ def WorkQueueSubmitThread(task_queue=multiprocessing.Queue(),
                 logger.error("Unable to create task: {}".format(e))
                 continue
 
+            if not autocategory or category is None:
```

Comment:
For a tiny bit more consistency you could say `autocategory is False`

Reply:
Yeah, that ended up looking awkward as things changed... Just fixed it